_haikus/chance_haiku.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Chance's Haiku
+author:Chance
+---
+
+Welcome to the cloud<br>
+It is now your home forever<br>
```

Comment:
This line is one syllable too long, you can amend it like this, for example:

```
It's now your home forever<br>
```

Reply:
Oh woops, my mistake.